tasks/config.yml

```diff
@@ -24,7 +24,23 @@
       group: "{{ teku_group }}"
     become: true
     register: config_toml
-  when: "not teku_standalone_validator | bool"
+  when:
+    - "not teku_standalone_validator | bool"
```

Comment:
more of a nit - I prefer the old school (a) and (b) so its really clear this is an `and` operation